packages/grid/src/utils/getGridClassNames.test.tsx

```diff
@@ -33,7 +46,15 @@ describe('Grid utils', () => {
     ).toEqual([
       'order-1_prefix',
       'width-auto_prefix',
-      'gutter-mobile-1_prefix',
+      undefined,
```

Comment:
As I understand we should avid `undefined `, why in this case is ok if result equal  to `undefined ` ?

Reply:
@derek-heycar In this test, we are testing dimensions, if anybody set any dimension less than "4" they can get "undefined", we have another task to improve `getGridClassNames` utils test, I'll open another pr after merging this pr

Reply:
But what is the meaning of this test case? 
Does not matter what you add this test will passed ?

Reply:
@derek-heycar Please check this pull request, I resolved this in this pr https://github.com/hey-car/heycar-uikit/pull/72/files